contents/docs/self-host/deploy/upgrade-notes.md

```diff
@@ -115,3 +115,60 @@ If you didn’t make any customization to those, there’s nothing you need to d
 
 - drops support for Kubernetes 1.19 as it has reached end of life on 2021-10-28
 - adds support for Kubernetes 1.23 released on 2021-12-07
+
+### Upgrading from 9.x.x
+
+10.0.0 introduces two major changes:
+
+1. as of today we've been including additional `StorageClass` definition into our Helm chart when installing it on AWS or GCP platforms. Starting from this release, we will not do that anymore and we will rely on the cluster default storage class. If you still want to install those additional storage classes, simply set `installCustomStorageClass: true` in your `values.yaml`.
+
+2. we have renamed few `values.yaml` file in order to reduce confusion and align our naming convention to the industry best practices:
+
+`clickhouseOperator.enabled` -> `clickhouse.enabled`
+`clickhouseOperator.namespace` -> `clickhouse.namespace`
+`clickhouseOperator.storage` -> `clickhouse.persistence.size`
+`clickhouseOperator.useNodeSelector` -> `clickhouse.useNodeSelector`
+`clickhouseOperator.serviceType` -> `clickhouse.serviceType`
+`clickhouse.persistentVolumeClaim` -> `clickhouse.persistence.existingClaim`
+
+If you are overriding any of those values, please make the corresponding changes before upgrading.
+
+<!---
```

Comment:
I'm hiding this as code comment as we need to wait for the next PostHog release before releasing it